benchmarks/trace/grpc/benchmark_test.go

```diff
@@ -0,0 +1,130 @@
+// Copyright 2017, OpenCensus Authors
```

Comment:
This should be under an internal directory.

/internal/bechmarks/grpc/trace/...